aries_vcx_core/src/errors/mapping_vdrtools.rs

```diff
@@ -1,11 +1,23 @@
-use vdrtools::types;
-use vdrtools::types::errors::IndyErrorKind;
+#[cfg(feature = "vdrtools")]
+use vdrtools::types::{
+    errors::{IndyError, IndyErrorKind},
+    ErrorCode,
+};
+
+#[cfg(all(
+    not(feature = "vdrtools"),
```

Comment:
wouldn't simply stating this one basically imply 
```
any(feature = "modular_libs", feature = "vdr_proxy_ledger")
```
? Cause if you don't build `vdrtools`, you must be building with something else, no?

I might be missing something

Reply:
Using your suggestion, if you build with `vdrtools` (default feat. flag) and one of `modular_libs` and `vdr_proxy_ledger`, your suggestion would fail due to duplicate import.